lib/attachinary/version.rb

```diff
@@ -1,3 +1,3 @@
 module Attachinary
-  VERSION = "1.3.0.5"
+  VERSION = "1.3.1"
```

Comment:
Should be `1.3.0.6` -- the 6 is how many times we've deviated from 1.3.0

Reply:
Gotcha, will change.